CHANGELOG.md

```diff
@@ -1,6 +1,24 @@
 # Changelog
+
+## 1.0.3: October 24, 2022
+
+– Updates related to implementing styles and patterns to better generalise site specific patterns and blocks. Admin documentation page update. ([DESCW-599]https://apps.itsm.gov.bc.ca/jira/browse/DESCW-599))
```

Comment:
if all of these are under  one ticket, maybe i can suggest something like work under ... jira ticket, and then just have the work in a sub list, adding the ticket behind each item, seems like too much work, and not required.